source/core/index.ts

```diff
@@ -118,6 +118,7 @@ type CacheableRequestFn = (
 ) => CacheableRequest.Emitter;
 
 type CheckServerIdentityFn = (hostname: string, certificate: DetailedPeerCertificate) => Error | void;
+export type ParseJsonFn = (text: string) => unknown;
```

Comment:
```suggestion
export type ParseJsonFunction = (text: string) => unknown;
```

I know we haven't been consistent with this, but never too late to start.

Reply:
Ok, I'm also going to change the other `*Fn` types, is it ok?